src/app/components/speakers/speakers.js

```diff
@@ -1,47 +1,43 @@
-
 import React from 'react';
 import { speakersData } from "assets/data/speakers-info.js";
 import './speakers.scss'
 
-const Speaker = (name, title, photo, index) => {
-    return (
-        <div className="speaker-img-wrapper">
-            <img 
-                src = {require('assets/images/speaker-photos/'+photo+'.jpeg')}
-                alt = {name + "'s profile picture"}
-            />
-            <div className="text">
-                <h4>{name}</h4>
-                <p>{title}</p>
-            </div>
-        </div>
-    );
-}
+export default function Speakers() {
 
-const renderSpeakers = (speakers) => {
-    return (
-        <div className="speaker-photos">
-            {speakers.map((speaker, index) => 
-                Speaker(speaker.name, speaker.title, speaker.photo, index)
-            )}
-        </div>
-    ) 
-}
+    function Speaker(name, title, photo, index) {
```

Comment:
Good job converting the Speaker component too!
- For function components where you only return a bunch of HTML, we can just skip the `return` keyword and give the stuff in the parentheses:
```JavaScript
const Speaker = (name, title, photo, index) => (
   <div ...>
      ...
   </div>
);
```
- Since `Speaker` is defined inside the `Speakers` component right now, it's a little inefficient because it'll be re-defined every time `Speakers` gets refreshed (so in case we have new data refreshing Speakers, but the Speaker component itself will always stay the same). We can just move it out to the lines right above `function Speakers(...)` so that it's defined separately, but `Speakers` can still use it to show each of the speakers